src/ngraph/attribute_adapter.cpp

```diff
@@ -201,6 +201,26 @@ namespace ngraph
         m_buffer_valid = false;
     }
 
+#ifdef __APPLE__
+    // size_t is not uint_64t on OSX
+    constexpr DiscreteTypeInfo AttributeAdapter<size_t>::type_info;
```

Comment:
Can we change size_t to uint64_t? It should help to avoid this code. Moreover for other platforms size_t also can have size which is not equal uint64_t.

Reply:
This works as long as size_t ins't bigger than int64_t. Hopefully we can get the size_ts changed before we hit 128-bit adddresses.